docs/5.community/5.framework-contribution.md

```diff
@@ -128,12 +128,13 @@ git checkout -b my-new-branch
 
 ### Set Up Documentation Website in Local Environment
 
-The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
+The Nuxt documentation is currently deployed within [nuxt/nuxt in `docs` folder](https://github.com/nuxt/nuxt/tree/main/docs).
```

Comment:
```suggestion
The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
```

Reply:
But actually, the url points to nowhere, the link is broken